RackSDK.cmake

```diff
@@ -0,0 +1,61 @@
+# Mapping of plugin build definitions from the Rack-SDK arch.mk, compile.mk, dep.mk and plugin.mk to CMake.
+
+set(RACK_SDK_VERSION 2.2.0)
+message(STATUS "load RackSDK.cmake (mapping based on Rack-SDK-${RACK_SDK_VERSION})")
+
+include_directories(${RACK_SDK_DIR}/include ${RACK_SDK_DIR}/dep/include)
+link_directories(${RACK_SDK_DIR})
+link_libraries(Rack)
+
+# This is needed for Rack for DAWs.
+# Static libs don't usually compiled with -fPIC, but since we're including them in a shared library, it's needed.
+add_compile_options(-fPIC)
+# Generate dependency files alongside the object files
+add_compile_options(-MMD -MP)
+# Debugger symbols. These are removed with `strip`.
+add_compile_options(-g)
+# Optimization
+add_compile_options(-O3 -funsafe-math-optimizations -fno-omit-frame-pointer)
+# Warnings
+add_compile_options(-Wall -Wextra -Wno-unused-parameter)
+# C++ standard
+set(CMAKE_CXX_STANDARD 11)
+
+if (${CMAKE_SYSTEM_NAME} MATCHES "Windows")
+  if (NOT MINGW)
+    message(FATAL_ERROR "Rack plugin development environment is only supported for MSYS2/MinGW")
+  endif ()
+  add_compile_definitions(ARCH_WIN ARCH_X64)
+  add_compile_definitions(_USE_MATH_DEFINES)
+  # disabled - causes surge-rack/surge/libs/sst/sst-plugininfra/src/misc_windows.cpp:20:20: error: cannot convert 'const char*' to 'LPCWSTR' {aka 'const wchar_t*'}
+  # see also https://gcc.gnu.org/onlinedocs/gcc/x86-Windows-Options.html
+  #add_compile_options(-municode)
```

Comment:
Here something seems to be not correct set in  https://github.com/surge-synthesizer/sst-plugininfra/blob/main/src/misc_windows.cpp#L19.